Readme.md

```diff
@@ -51,17 +50,16 @@ Supported units and abbreviations are as follows and are case-insensitive:
 | 1024<sup>7</sup> | ZiB  | zebibite  |
 | 1024<sup>8</sup> | YiB  | yobibite  |
 
+### Compatibility Binary (JEDEC)
 
-### Compatibility Binary Prefixes (JEDEC)
-
-Overwrites the lower units of the metric system with the commonly misused prefixes
+Overwrites the lower units of the metric system with the commonly misused values. The higher units are not supported.
 
 |      Value       | Abbr |   Name    |
 | ---------------- | ---- | --------- |
-| 1000<sup>1</sup> | kB   | kilobyte  |
-| 1000<sup>2</sup> | MB   | megabyte  |
-| 1000<sup>3</sup> | GB   | gigabyte  |
-| 1000<sup>4</sup> | TB   | terabyte  |
+| 1024<sup>1</sup> | kB   | kilobyte  |
```

Comment:
lol yeah surprised no-one else caught this

Reply:
I have a frustrating tendency to be the first to report minor issues ;)

Reply:
this isn't really minor! so thank you